src/PhpSpec/Formatter/ProgressFormatter.php

```diff
@@ -87,7 +87,7 @@ function ($count) use ($total) {
     private function getBarLengths($percents)
     {
         $stats = $this->getStatisticsCollector();
-        $specProgress = ($stats->getTotalSpecs()+1)/$stats->getTotalSpecsCount();
+        $specProgress = $stats->getTotalSpecs() ? ($stats->getTotalSpecs()+1)/$stats->getTotalSpecsCount() : 0;
```

Comment:
Technically, we're dividing by `totalSpecsCount` here, not `totalSpecs`. I'd rather check if `getTotalSpecsCount()` returns something that's not a zero. If there was an error in the statistics collector, this could still fail. By using `totalSpecs` instead of `totalSpecsCount`, `ProgressFormatter` makes assumptions on how statistics collector works.

Also, formatting is not consistent with the rest of the code. More spaces please ;)

```php
$specProgress = $stats->getTotalSpecsCount() ? ($stats->getTotalSpecs() + 1) / $stats->getTotalSpecsCount() : 0;
```